1-mandatory/3-slideshow/slideshow.js

```diff
@@ -1 +1,148 @@
 // Write your code here
+
+/*
+1. Put the images on top of each other using CSS.
+2. Load one image at a time.
+    step 1: create an arrary of the image sources
+    step 2: load the first image source to the image tag (when the website uploads using JS we can have our first image displayed on the page using the first array index)
+    step 3: We can make the image go forward by incrementing index postion (go up by 1) when we click "forward"
+    step 4: We can make the image go backward by decreasing the index postion (go down by 1) when we click the "backward"
+*/
+
+
+const images = [
+  {
+    src: "./assets/cute-cat-a.png",
+    alt: "cute white cat with black and grey stripes",
+    caption: "cute cat",
+  },
+  {
+    src: "./assets/cute-cat-b.jpg",
+  },
+  {
+    src: "./assets/cute-cat-c.jpg",
+  },
+  // {
+  //   src: "./assets/cute-cat-d.jpg",
+  // },
+];
+
+const image = document.querySelector("#carousel-img");
+const autoForwardBtn = document.querySelector("#auto-forward");
+const forwardBtn = document.querySelector("#forward-btn");
+const stopBtn = document.querySelector("#stop");
+const backwardBtn = document.querySelector("#backward-btn");
+const autoBackBtn = document.querySelector("#auto-backward");
```

Comment:
I like the structure of declaring elements in one place, much easier to read code this way! Nicely done